testsuite/MDAnalysisTests/topology/test_fetch_pdb.py

```diff
@@ -0,0 +1,125 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the Lesser GNU Public Licence, v2.1 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+from urllib import request
+
+import MDAnalysis as mda
+import pytest
+
+
+def has_pooch():
+    try:
+        import pooch
+        from requests.exceptions import HTTPError
+
+        return True
+    except ModuleNotFoundError:
+        return False
+
+
+def has_internet():
+    try:
+        request.urlopen("https://files.wwpdb.org/", timeout=2)
+        return True
+    except request.URLError:
+        return False
+
+
+@pytest.mark.skipif(
+    not has_pooch() or not has_internet(),
+    reason="Cannot connect to https://files.wwpdb.org/",
+)
+class TestDocstringExamples:
+    """This class tests all the examples found in fetch_pdb's docstring"""
+
+    TRUE_NATOMS_PER_PDB = {"1AKE": 3816, "4BWZ": 2824}
```

Comment:
When you're testing multiple data with the same code, you should learn to use paramtrized tests in pytest https://docs.pytest.org/en/stable/how-to/parametrize.html

`@pytest.mark.parametrize("pdb_id,n_atoms", [("1AKE", 3816), ("4BWZ", 2824)])` might do the job. Then your tests can use
```python
def test_xxxx(self, tmp_path, pdb_id, n_atoms):
   p = mda.fetch_pdb(pdb_id, cache_path=tmp_path, file_format="cif")
   u = mda.Universe(p)
   assert u.atoms.n_atoms == n_atoms, f"Incorrect number of atoms for {pdb_id}"
```